packages/@cdklabs/typewriter/src/renderer/typescript.ts

```diff
@@ -57,13 +57,29 @@ import { PrimitiveType, Type } from '../type';
 import { TypeParameterSpec } from '../type-declaration';
 import { Initializer, MemberVisibility, Method } from '../type-member';
 
+export interface TypeScriptRenderOptions extends RenderOptions {
+  eslintPrettier?: boolean;
+  eslintCommaDangle?: boolean;
+}
+
 export class TypeScriptRenderer extends Renderer {
+  private eslintPrettier: boolean;
+  private eslintCommaDangle: boolean;
+  public constructor(options: TypeScriptRenderOptions = {}) {
+    super(options);
+
+    this.eslintPrettier = options.eslintPrettier ?? false;
+    this.eslintCommaDangle = options.eslintCommaDangle ?? true;
```

Comment:
If we do this, should we just go all in and take an array of eslint rules to disable?